ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/TransactionCommitterSpec.scala

```diff
@@ -207,6 +217,91 @@ class TransactionCommitterSpec extends AnyWordSpec with Matchers with MockitoSug
         }
       }
     }
+
+    "setting dedup context" should {
+      val deduplicateUntil = protobuf.Timestamp.newBuilder().setSeconds(30).build()
+      val submissionTime = protobuf.Timestamp.newBuilder().setSeconds(60)
+
+      def buildContextAndTransaction(
+          submitterInfoAugmenter: DamlSubmitterInfo.Builder => DamlSubmitterInfo.Builder
+      ) = {
+
+        val context = createCommitContext(None)
+        context.set(Conversions.configurationStateKey, aDamlConfigurationStateValue)
+        val transactionEntrySummary = DamlTransactionEntrySummary(
+          aDamlTransactionEntry.toBuilder
+            .setSubmitterInfo(
+              submitterInfoAugmenter(
+                DamlSubmitterInfo
+                  .newBuilder()
+              )
+            )
+            .setSubmissionTime(submissionTime)
+            .build()
+        )
+        context -> transactionEntrySummary
+      }
+
+      def contextSetDeduplicateUntil(
+          context: CommitContext,
+          transactionEntrySummary: DamlTransactionEntrySummary,
+      ) = {
+        context
+          .get(Conversions.commandDedupKey(transactionEntrySummary.submitterInfo))
+          .value
+          .getCommandDedup
+          .getDeduplicatedUntil
+      }
```

Comment:
Braces not needed

Reply:
Can we add this to scalafmt? I have no preference in the `with brace`/`without braces` skirmish but I would very much prefer the formatter to handle code styling like this and not have to do it manually.

Reply:
(It's really a nit so not very important) Idea unfortunately adds those automatically when extracting a method; I think some form of control is available in `scalafmt` according to the [docs](https://scalameta.org/scalafmt/docs/configuration.html), probably around redundant braces.